build-test.xml

```diff
@@ -30,6 +30,7 @@
         </fileset>
         <fileset dir="${java.home}/../lib">
             <include name="**/*.jar" />
+            <exclude name="missioncontrol/**" />
```

Comment:
This excludes a whole bunch of extra eclipse libs and stuff that we don't use from the Oracle 8 JDK.  (Which IDEA doesn't use now, but, according to JetBrains, will be required for IDEA 16 releases.)  It's not necessary, but I left it in place because that cost me about two hours work.